.github/workflows/build.yml

```diff
@@ -28,5 +28,6 @@ jobs:
         cache: 'npm'
     - run: npm ci
     - run: npm run build --if-present
+    - run: npm run build:cli
```

Comment:
Could use an update to the readme, or a readme in the CLI directory explaining what the cli does with a few examples.